docs/conf.py

```diff
@@ -40,18 +40,14 @@
 # ones.
 extensions = [
     'notfound.extension',
-    # 'autoapi.extension',
+    'autoapi.extension',
     'sphinx_tabs.tabs',
     'sphinx-prompt',
     'sphinxemoji.sphinxemoji',
 ]
 
-notfound_context = {
```

Comment:
We are using the default context now, right?

Reply:
Yes.